image.yaml

```diff
@@ -40,9 +40,13 @@ modules:
       install:
           - name: datagrid
 artifacts:
-    - path: jboss-datagrid-7.2.0-server.zip
-      description: "Artifact is available on Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=50851&product=data.grid&version=&downloadType=distributions"
-      md5: 411376fedbd0df3ed373ae56e0a69c3c
+    # Uncomment this once we are at CR/Final phase
+    # - path: jboss-datagrid-7.2.0-server.zip
+    #   description: "Artifact is available on Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=50851&product=data.grid&version=&downloadType=distributions"
+    #   md5: 411376fedbd0df3ed373ae56e0a69c3c
+    - path: infinispan-server-8.5.0-redhat-SNAPSHOT-147-bin.zip
+      description: "Development artifact. Will be replaced by CR or Final version soon"
+      md5: e4913e57e6421da97bb7ad465e079786
```

Comment:
The ```Dockerfile``` supports a directive called ```ARG```, which is a variable that you [can pass to the docker build command](https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables-build-arg). This would allow to parametrise the build without having to have commented snippets. 
You'd need to check if the tool that generates the Dockerfile from this yaml supports this though

Reply:
That's a neat idea but it won't work here. `Concreate` also puts artifacts in `/tmp/artifacts` and we need to tell it whether we want the official/final archive (which would be downloaded from Customer Portal) or a development one (which would be downloaded from our Jenkins).